### DISCUSSION TITLE
One big folder for all plots

### DISCUSSION THREAD

**Discussion by hajes** [Chia-Network/chia-blockchain#14963] [General]
The idea was to consolidate all mount points to one virtual directory instead of changing `config.yaml` all the time.

I have used symlinks `ln -s /mnt-XX/*.plots /chia-plots` from nine mount points to a single directory.

All seems ok with `chia farm summary`. All plots there but no proof sent to pool.

Do you guys have some safety feature that refuses to follow symlinks to real plot?

Does harvester recursively search subdirs?

I have now `/chia-plots/mnt/c1-9`. if I type in `config.yaml` just `/chia-plots`, would be everything found recursively, please?

**Comment by Ganbin** [comment]
Hi,

You don't need to do that, in `config.yaml`, you can set a single `plot_directories` to `/mnt` for example and then set `recursive_plot_scan: true`.

This will recursively look all your directories in `/mnt`.

In your case, I see that you have different roots, I would recommend that you mount your nine drives in a single root. Ex. `/mnt/hdd1`, `/mnt/hddXX`, etc.

**Reply by hajes** [reply]
thanks for info. Unfortunately, I have the same `config.yaml` since the beginning, and they do not update with new version. I didn`t find this option :-( Will do, it would simplify everything.

**Reply by hajes** [reply]
`config.yaml` it would be great if you guys post reference/all options on Wiki ;-)

**Reply by Ganbin** [reply]
You can find them here: [initial-config.yaml](https://github.com/Chia-Network/chia-blockchain/blob/main/chia/util/initial-config.yaml)

**Reply by hajes** [reply]
I have accidentally found this file about a month ago. Thanks anyway Ganbin

**Reply by hajes** [reply]
Is there a way to exclude subdirectories from `recursive_plot_scan`, please?

I have two types of plots, and of course, it spams log with ERRORs about `invalid plot format`

**Reply by Ganbin** [reply]
I don't think so, but I suggest you to have a different folder for the 2 different types of plots. Like `/mnt/plots1` and `/mnt/plots2`. You would then use `plot_directories: /mnt/plot1`. It will then only recursively look into `/mnt/plot1`
